src/index.js

```diff
@@ -20,6 +20,9 @@ function serializeComponents(props) {
     } else if (props[component].constructor === Object) {
       // Stringify components passed as object.
       serialProps[component] = styleParser.stringify(props[component]);
+    } else if (props[component] === true) {
```

Comment:
hmm what happens if there's a component that accepts a boolean value that defaults to false.

something like ```visible={true}```, which would still work in this case since it defaults to true, but it's possible for a component to accept bool